packages/ui-test-sandbox/src/utils/reactComponentWrapper.js

```diff
@@ -53,11 +53,6 @@ class ReactComponentWrapper {
               wrapper.setChildProps(newProps)
             })
           },
-          setContext(newContext) {
-            return doAsync(() => {
-              wrapper.setChildContext(newContext)
-            })
-          },
```

Comment:
This was setting legacy context and was used only in one test, which I just refactored